### ISSUE TITLE
Placeholder - Prep for Email confirmation

### ISSUE THREAD

**Issue by randyabramson** [usdoj-crt/crt-portal-management#1027]
Notes from Story time

1. Limit to report number, what to expect next and what you can do next (don't include full report)

2. Need translations and adjust copy

3. Need to remove 'you will not get an confirmation' copy on last page on site

4. On 'when you will hear back' we are thinking of 'we try to get responses back within 30 days, but there are times that it takes longer or you might not hear from us at all'

5. Emily is voting that we auto reply with confirmation (no button that says 'send me a confirmation')

Once auto confirm, we will turn of Email replies to ADM box (no reply)

**Comment by emilycryan** [comment]
Here's my proposed email copy. I have not included any formatting - if it's at all possible to even get underline and bold, that would be very helpful here. It's a bit bland right now. 

==================================================
Please do not reply to this email. This is an unmonitored account.
==================================================

Thank you for submitting a report to the Civil Rights Division. Please save your record number for tracking. Your record number is: 888-MPL.

---------------------------------

What to expect
1. We review your report
Our specialists in the Civil Rights Division carefully read every report to identify civil rights violations, spot trends, and determine if we have authority to help with your report.

2. Our specialists determine the next steps
We may decide to:
* Open an investigation or take some other action within the legal authority of the Justice Department.
* Collect more information before we can look into your report.
* Recommend another government agency that can properly look into your report. If so, we’ll let you know.
In some cases, we may determine that we don’t have legal authority to handle your report and will recommend that you seek help from a private lawyer or local legal aid organization.

3. When possible, we will follow up with you We do our best to let you know about the outcome of our review. However, we may not always be able to provide you with updates because:
* We’re actively working on an investigation or case related to your report.
* We’re receiving and actively reviewing many requests at the same time.
If we are able to respond, we will contact you using the contact information you provided in this report. Depending on the type of report, response times can vary. If you need to reach us about your report, please refer to your report number when contacting us. This is how we keep track of your submission.

---------------------------------

What you can do next
1. Contact local legal aid organizations or a lawyer if you haven’t already
Legal aid offices or members of lawyer associations in your state may be able to help you with your issue.
* American Bar Association, visit www.findlegalhelp.org or call (800) 285-2221
* Legal Service Corporation (or Legal Aid Offices), visit www.lsc.gov/find-legal-aid or call (202) 295-1500

2. Get help immediately if you are in danger
If you reported an incident where you or someone else has experienced or is still experiencing physical harm or violence, or are in immediate danger, please call 911 and contact the police.

---------------------------------

PLEASE NOTE: Each week, we receive hundreds of reports of potential violations.  We collect and analyze this information to help us select cases, and we may use this information as evidence in an existing case.  We will review your letter to decide whether it is necessary to contact you for additional information.  We do not have the resources to follow-up on every letter.

**Comment by randyabramson** [comment]
This really great @emilycryan! I don't see any changes that need to be made.

One question (and I realize this is a big ask of the user...especially since they already submitted a report), in the 'what you can do next' section, should we have a section that says:

"Learn more about your rights and see examples of violations we handle" and link to https://civilrights.justice.gov/#your-rights

I know this is a long shot idea, but I'd like to try to get folks to start educating themselves a bit more. Who knows...maybe it will lead to fewer repeat fliers?

Lastly...the next step would be to think about UI changes we need in the form to account for this auto confirmation (maybe a new card?):

- [ ] Update language on nudge/contact info screen
- [ ] Update language on confirmation screen
- [ ] Where else?

**Comment by emilycryan** [comment]
@randyabramson We can't do links to content in plain text, unfortunately, so I wouldn't recommend including that link as-is. It's too long/complicated to type out. 

I'm not sure we need content updates on the nudge/contact screen - what were you thinking here? For the final screen, I'd keep it simple and say "Print or save this page for your records." 

CC @blaughman

**Comment by randyabramson** [comment]
Ah yes, @emilycryan  - wishful thinking on my part. We could say 'Visit civilrights.justice.gov to learn more about your rights and see examples of violations we handle" 

(Always pushing the education :))

**For contact info page would we add:**

You are not required to provide your name or contact information. If you want to remain anonymous, leave this section blank. If you choose to provide your contact information, we will only use it to respond to your submission. **Adding your Email address will allow us to send you a confirmation after submitting your report.**

**On confirmation page, would we add:**

**"If you provided a valid Email address, you will receive a confirmation."**

**Comment by emilycryan** [comment]
@randyabramson We can add the education piece to the email if you think it'll help.

As for this "Adding your Email address will allow us to send you a confirmation after submitting your report."  I would say that saying they're going to get a confirmation email is sort of a moot point - they're already expecting one. It's just more text to read and add to the popup content we already show. In addition, it actually brings more awareness to something that should be seamless so if there's even the slightest possibility that the email will not go out bc gov delivery is down, you've potentially caused more issues bc they're now actively expecting something. To me, it's not adding much in the way of value and is just more words to read. Same holds true re. "If you provided a valid Email address, you will receive a confirmation." If we draw attention to it, and it doesn't happen, it'll likely cause more issues for the user. CC @blaughman

**Comment by randyabramson** [comment]
@emilycryan  @blaughman 

Next steps:


- [ ] Finalize Auto response copy 

- [ ] look at changes that need to happen on site

- [ ] consider translations

**Comment by emilycryan** [comment]
Ok, updated version of the email. LMK how this looks:

![Screen Shot 2021-09-02 at 3 18 56 PM](https://user-images.githubusercontent.com/73443807/131903894-7f6496af-d396-468d-a146-49bf61d22f08.png)

Text versions:
[Auto-reply email (1).txt](https://github.com/usdoj-crt/crt-portal-management/files/7101245/Auto-reply.email.1.txt)
[Auto-reply email (1).docx](https://github.com/usdoj-crt/crt-portal-management/files/7101246/Auto-reply.email.1.docx)
[Auto-reply email.pdf](https://github.com/usdoj-crt/crt-portal-management/files/7101247/Auto-reply.email.pdf)

CC @blaughman @randyabramson

**Comment by randyabramson** [comment]
Wow... @emilycryan  this looks soooooo goood! Seeing this really makes all of the stuff that we have been talking about around public trust more real for me!

Some things we should discuss in a meeting:

1. Who might need to review this (Front office? Select intake specialists?)
2. How do we handle translations?
3. Is first slice English only (my instinct is no, but let's discuss)
4. Front end changes:
- Do we mention on contact area OR contact nudge that valid email address will help with report confirmation messaging?
- Confirmation page currently says 'You will not receive an email confirmation, so please print or save this page for your records.'
5. Anything else?

I'll set up a meeting to discuss.

cc @blaughman

**Comment by randyabramson** [comment]
Next steps:

1. Get email approved by Dan
2. Get email translated
3. Review notes on above (Randy has notes)

**Comment by emilycryan** [comment]
Ok, with Dan's edits. Text files below screenshot of the email. NOTE: Email has no formatting in it in this first round (ignore the yellow phone numbers) CC @randyabramson @blaughman @billyhunt @ilodidoj @louh 

![Screen Shot 2021-09-07 at 12 40 33 PM](https://user-images.githubusercontent.com/73443807/132381042-a1e46ff2-e880-405f-a2d2-6039cf5f75d7.png)

[Auto-reply email.pdf](https://github.com/usdoj-crt/crt-portal-management/files/7123101/Auto-reply.email.pdf)
[Auto-reply email (2).txt](https://github.com/usdoj-crt/crt-portal-management/files/7123102/Auto-reply.email.2.txt)
[Auto-reply email (2).docx](https://github.com/usdoj-crt/crt-portal-management/files/7123103/Auto-reply.email.2.docx)

**Comment by randyabramson** [comment]
@emilycryan @blaughman what do you think of Henry's version here. It certainly is shorter!

[Auto-reply.email-Henry-v2.docx](https://github.com/usdoj-crt/crt-portal-management/files/7123154/Auto-reply.email-Henry-v2.docx)

**Comment by emilycryan** [comment]
@randyabramson "If we think that you should go to another agency or a private lawyer, we’ll let you know." This feels like a problematic sentence. And repetitive from the bullets just above it. 

Is formatting something we will do? Because this version only works with bold and bullets. It also feels oddly almost **_too short_** but perhaps that's just me. CC @blaughman

**Comment by randyabramson** [comment]
@emilycryan in the previous version (and what's on the site), we say 'In some cases, we may determine that we don’t have legal authority to handle your report and will recommend that you seek help from a private lawyer or local legal aid organization.'

@blaughman do you wanna weigh in?

**Comment by emilycryan** [comment]
@randyabramson Agreed but the way it's worded in Henry's version sounds pretty different to me than what's on the site. We say "in some cases... we may ... will recommend" vs "If we think that you should go to another agency or a private lawyer, we’ll let you know." I can imagine people will read the second version _differently_. To me the first feels more like potential. The second feels like a promise. And we cannot promise that we will individually get back to each person on whether or not they should seek legal counsel. 

To clarify, they are SUPER similar but having heard the volume of feedback around assuming that DOJ is acting as legal representation, I'd be very careful about any wording that makes that feel more like an obligation on our part. It's not and we shouldn't give any indication by using vague wording that it would become part of our role. CC @blaughman

**Comment by randyabramson** [comment]
BTW @emilycryan @blaughman DEVS say bullets might not be supported, so we would have to rely on spacing and dashes. The HCE letter is a good one to look at.

**Comment by emilycryan** [comment]
Do we know when they will say either way? (You said they said "might") We can utilize other methods like dashes if need be. CC @randyabramson @blaughman

**Comment by randyabramson** [comment]
@billyhunt  said he doesn't recommend it because you don't know what client folks have, so let's go with "NO"

**Comment by randyabramson** [comment]
@emilycryan You made a great point above. I went through the 2 versions of the letters side by side and made some tweaks/additions to Henry's version below. Thoughts? cc: @blaughman 

[Auto-reply.email-Henry-RA-Edit-v2.docx](https://github.com/usdoj-crt/crt-portal-management/files/7128815/Auto-reply.email-Henry-RA-Edit-v2.docx)

**Comment by blaughman** [comment]
@randyabramson @emilycryan  Can we devote the last 10 minutes of standup today to discuss the content?  Randy - the line in What's next that says, "We also might not..." would seem confusing to me as a recipient.  So, you think my allegations are noteworthy, but you won't get back to me.  I feel this needs more context, but I'm not sure we keep to our simple, straightforward approach if we try to flesh this out.  Just my .02.

**Comment by emilycryan** [comment]
I feel like perfect is the enemy of good enough. The first version of the reply email was basically the thank you/submission page. We originally said it was worth sending that out b/c most individuals no longer had access to the page after closing their browser and leaving it. We are now trying to wordsmith the same general meaning but with less words (?!?) 

Basically, if we adhere to how we normally like to work, it's "thin slice" to use @randyabramson terminology. Our thin slice was getting that last page of content into a non-designed email and creating an auto-responder. I feel like we should do that and leave the content as-is for now and maybe iterate on future versions. Largely because a) we wanted to see if a responder alone would cut down on followups to sections, b) we still don't know exactly how to handle the multi-language needs and c) we don't even know yet if this will all work from a server reliability perspective. We are basically overcomplicating this.

Perhaps we just leave the email as-is (with Dan's suggestion to move immediate danger up) and launch with that first...? This is getting down a huge rat hole and there's no really good reason to IMO if the goal is just to see what happens initially. 

CC @blaughman

**Comment by emilycryan** [comment]
Also, these new versions have nested bullets within bullets which, if we're getting rid of bullets, won't work. The language and content will not come through as expected and it'll be a confusing mess. CC @randyabramson @blaughman

**Comment by blaughman** [comment]
Spot on @emilycryan.  Let's keep this simple and consistent with the current experience and iterate, if needed, by monitoring Touchpoints and the AskCRT inbox.

cc @randyabramson

**Comment by emilycryan** [comment]
OK, here's the original, no bullets or special formatting (and ignore the yellow text which I cannot turn off for the life of me) I think this feels solid. CC @randyabramson @blaughman 

![Screen Shot 2021-09-08 at 9 23 19 AM](https://user-images.githubusercontent.com/73443807/132517661-ef2c50eb-ea93-4e8c-bfbc-6745e931ead3.png)
